onnx/backend/test/cpp/driver/test_driver.h

```diff
@@ -32,28 +32,28 @@ struct TestData {
  *	No real data was loaded in this type of class,
  *	but only data location.
  */
-struct TestCase {
-  TestCase(
+struct UnsolvedTestCase {
+  UnsolvedTestCase(
       const std::string& model_filename,
       const std::string& model_dirname,
-      const std::vector<TestData>& test_data)
+      const std::vector<UnsolvedTestData>& test_data)
       : model_filename_(model_filename),
         model_dirname_(model_dirname),
         test_data_(test_data) {}
 
-  TestCase() {}
+  UnsolvedTestCase() {}
 
   std::string model_filename_;
   std::string model_dirname_;
-  std::vector<TestData> test_data_;
+  std::vector<UnsolvedTestData> test_data_;
 };
 
 /**
  *	Store one chunk of test data,
  *	including raw input/output and protos
  *	Real data was loaded in this type of class..
  */
-struct ProtoTestData {
+struct ResolvedTestData {
```

Comment:
can you comment what Unsolved/Resolved actually mean here :) in the comment for the classes?

Reply:
Sure!